src/sonic-host-services/scripts/caclmgrd

```diff
@@ -406,6 +418,15 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
                             self.log_warning("rule_props for rule_id {} empty or null!".format(rule_id))
                             continue
 
+                        rule_props = {k.upper(): v for k,v in rule_props.items()}
+
+                        ethertype = 0
+                        if "ETHER_TYPE" in rule_props:
+                            ethertype = get_ethertype_from_rule_props(rule_props)
+                            if ethertype != 0x800 and ethertype != 0x86dd:
+                                log_error("rule_props for rule_id {} does not have valid/supported ethertype.".format(rule_id))
+                                continue
```

Comment:
Suggest defining constants at the top of the file like,
```python3
ETHERTYPE_UNKNOWN = 0x0
ETHERTYPE_IPV4 = 0x0800
ETHERTYPE_IPV6 = 0x86dd
```

Suggest simplifying this as either:

```python
if "ETHER_TYPE" in rule_props:
    if get_ethertype_from_rule_props(rule_props) not in [ETHERTYPE_IPV4, ETHERTYPE_IPV6]:
        log_error("rule_props for rule_id {} does not have valid/supported ethertype.".format(rule_id))
        continue
```

or

```python
if get_ethertype_from_rule_props(rule_props) not in [ETHERTYPE_UNKNOWN, ETHERTYPE_IPV4, ETHERTYPE_IPV6]:
    log_error("rule_props for rule_id {} does not have valid/supported ethertype.".format(rule_id))
    continue
```

And suggest using these constants on lines 287 and 296 also.